components/icon/stories/template.js

```diff
@@ -171,9 +171,9 @@ export const FullIconSetTemplate = (args, context) => {
 			})}
 		>
 			${when(args.setName === "workflow", () => {
-				return workflowIconsCleaned.sort().map((iconName) => IconWithLabelTemplate({ ...args, iconName }, context));
+				return workflowIconsCleaned.map((iconName) => IconWithLabelTemplate({ ...args, iconName }, context));
 			}, () => {
-				return uiIconsWithDirections.sort().map((iconName) => IconWithLabelTemplate({ ...args, uiIconName: iconName }, context));
```

Comment:
I believe these two were already being sorted in a way that separates the icon name and numerical size, add `.sort()` here was reverting the sorted icons list back to ASCII order (so that `Add100` appears before `Add50`, for instance).